lib/parser/resolve-packages.js

```diff
@@ -0,0 +1,36 @@
+const transform = require('./transform'); //eslint-disable-line
+const {spawn} = require('child_process');
+const path = require('path');
+
+function processPromise(child) {
+	return new Promise(function(resolve, reject) { //eslint-disable-line
+		child.addListener('error', reject);
+		child.addListener('exit', resolve);
+	});
+}
+function spawnChild(pkg) {
+	// spawn('npm', ['install', '--save', pkg], { stdio: 'inherit', customFds: [0, 1, 2] });
+	return spawn('npm', ['install', '--save', pkg]);
+}
+
+module.exports = function resolvePackages(option) {
+	option.packages.filter( (pkg) => {
+		processPromise(spawnChild(pkg)).then( (result) => {
```

Comment:
We should not have lintin errors merged into master, In other parts we are doing `eslint-disable` for issues that are happening now, but as these ones, will be fixed later when we have more stuff in